docs-new/docs/graphql/core/databases/postgres/mutations/update.mdx

```diff
@@ -392,8 +393,8 @@ The currently available `jsonb` operators are:
 
 | Operator          | Postgres equivalent | Function                                     |
 |-------------------|---------------------|----------------------------------------------|
-| `_append`         | \| \|                | append json value to a `jsonb` column        |
-| `_prepend`        | \| \|                | prepend json value to a `jsonb` column       |
+| `_append`         | `||`                | append json value to a `jsonb` column        |
+| `_prepend`        | `||`                | prepend json value to a `jsonb` column       |
```

Comment:
My bad, looks like this is not parsing correctly.

Apologies for causing confusion.

Lets rollback to using escape chars `\|\|` while removing the sapce between them.

Please verify in UI to see if this is rendering correctly.